components/CredentialField.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div v-if="show">
+    <q-item class="col q-py-none">
+      <q-item-section>
+        <q-item-label
+          v-if="title"
+          :class="titleClass"
+          :lines="lines">
+          {{title}}
+        </q-item-label>
+        <q-item-label
+          :class="valueClass"
+          :lines="lines">
+          {{value}}
+        </q-item-label>
+      </q-item-section>
+    </q-item>
+  </div>
+</template>
+
+<script setup>
+import {computed, defineProps} from 'vue';
+
+const props = defineProps({
+  title: {
+    type: String,
+    default: ''
+  },
+  value: {
+    type: String,
+    default: ''
+  },
+  titleClass: {
+    type: String,
+    default: 'text-caption text-center text-grey-7'
+  },
+  valueClass: {
+    type: String,
+    default: 'text-body2 text-center'
+  },
+  lines: {
+    type: String,
+    default: '2'
+  }
+});
+
+const show = computed(() => {
+  return props.value.length > 0;
+});
```

Comment:
Could be simplified:

```suggestion
const show = computed(() => props.value.length > 0);
```

Reply:
4a1122145b9b4b1c9023ab51062930e14c2fe365